lib/rubygems/installer.rb

```diff
@@ -86,11 +84,6 @@ def inherited(klass)
       super(klass)
     end
 
-    ##
-    # True if we've warned about PATH not including Gem.bindir
-
-    attr_accessor :path_warning
```

Comment:
Should I gracefully deprecate this accessor? I searched in public GitHub code and did not see anything, but I did not use gem-codesearch. @hsbt do you have that setup to check?

Reply:
Looking at https://github.com/search?q=%22Gem%3A%3AInstaller.path_warning%22&type=code&p=1 it seems like it's all generated documentation, a 13-year-old copy of openbsd-ports, and copies of the Ruby or RubyGems repos.

I don't think we need to worry about it.

Reply:
Cool, let's merge it then, I did the same search and observed the same thing 👍.